hack/bin/template-chart.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+sed -i "s/\[\[\ \.Version\ \]\]/2.1.1/g" helm/prometheus-rules/Chart.yaml
+helm template helm/prometheus-rules --output-dir hack/output
+sed -i "s/2\.1\.1/\[\[\ \.Version\ \]\]/g" helm/prometheus-rules/Chart.yaml
```

Comment:
We can kill this by re-using what we already have in the `test/hack/bin/verify-rules.sh`, where the helm chart is being templated already https://github.com/giantswarm/prometheus-rules/blob/7f29bdc3ff41fcdfb7e2edf5ea995543f9dd8b35/test/hack/bin/verify-rules.sh#L38
We can extract this line into the Makefile so your script also benefit from it.

Reply:
I have prepared a PR which does this https://github.com/giantswarm/prometheus-rules/pull/580

Reply:
Makefile updated